app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -1273,14 +1197,18 @@ class AdvancedFilter extends React.Component {
                     this.changeCombinationValue(statementIndex, combinationIndex, { name: combinationValue.name, value: event.target.value });
                   }}>
                   {combinationFilter.options.map((option, o_index) => {
-                    return <option key={o_index}>{option}</option>;
+                    return (
+                      <option key={o_index} value={option}>
+                        {option}
+                      </option>
+                    );
                   })}
                 </Form.Control>
               )}
               {combinationFilter?.type === 'search' && (
                 <Form.Control
                   value={combinationValue.value}
-                  className="advanced-filter-combination-multi-select-options advanced-filter-multi-select-options my-0 mx-3 py-0"
+                  className="advanced-filter-combination-search-input my-0 mx-3 py-0"
```

Comment:
The width of the input is reduced compared with `1.44.0`
<img width="1118" alt="Screen Shot 2022-03-22 at 7 53 48 PM" src="https://user-images.githubusercontent.com/2328582/159596368-a8b8013a-d8ce-45d4-b52a-4039bcead1e3.png">
<img width="1114" alt="Screen Shot 2022-03-22 at 7 52 43 PM" src="https://user-images.githubusercontent.com/2328582/159596403-2ef7a7b4-f3c8-4bec-8f02-46c1b7a1ac1d.png">

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1239/commits/5907261b523cee88d902c175c0e9d67c52b53579